### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by kalbek**
## Changes made in this PR:
- Create Routes:
   - ```users```
   - ```users/:id```
   - ```users/:user_id/posts```
   - ```users/:user_id/posts/:id```
- Create controllers with correct actions
- Create views for all routes with plain HTML

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# app/controllers/posts_controller.rb
+class PostsController < ApplicationController
```
Comment (brytebee) [inline app/controllers/posts_controller.rb:2]: An index action is required for this controller.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# app/controllers/users_controller.rb
+class UsersController < ApplicationController
```
Comment (brytebee) [inline app/controllers/users_controller.rb:2]: An index action is required for this controller.

File: config/routes.rb [hunk 3]
```diff
@@ -3,4 +3,16 @@
...
+
+  # Homepage route
+  get 'users', to: 'home#index', as: :users
+
+  # User profile route
+  get 'users/:id', to: 'users#show', as: :user
+
+  # User posts route
+  get 'users/:user_id/posts', to: 'posts#index', as: :user_posts
+
```
Comment (brytebee) [inline config/routes.rb:15]: Kindly implement a corresponding route for the user & post indexes previously mentioned.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @kalbek 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great understanding and implementation of MVC.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @kalbek, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- Gitflow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**